Repositories/IListingRepository.cs

```diff
@@ -7,6 +7,10 @@ public interface IListingRepository
     {
         IEnumerable<Listing> GetAllListings();
 
+        IEnumerable<Listing> GetAllListings(string id);
```

Comment:
The name of this method is unclear a.k.a. which id is used here. I would suggest to rename to GelAllUserListings or something and for parameter userId or ownerId.